rust/src/lib/ifaces/linux_bridge.rs

```diff
@@ -367,7 +394,15 @@ impl LinuxBridgeConfig {
 #[non_exhaustive]
 pub struct LinuxBridgePortConfig {
     /// The kernel interface name of this bridge port.
+    /// When applying with both `profile-name` and `name` defined, only
+    /// `profile-name` will take effect.
```

Comment:
Can we add tests to validate the situation that both `profile-name`  and `name` are defined.